### PR TITLE
feat(type): system font stack

### PR DESCRIPTION
**Pull request by juliodialpad**
## Description

We are moving from Inter to using a system font stack.

- Replaced `@define-ff-custom-name` value with new font stack
- Removed `@ff-inter` and `@ff-inter-italic` variables
- Removed Inter from font declaration
- Removed Inter font files from `font` folder
- Added Adjusted Segoe font

### Changes noticed

| Before | After |
:-------------------------: | :-------------------------:
Screen Shot 2022-09-09 at 12 02 42 p m | Screen Shot 2022-09-09 at 12 02 54 p m
image | image
image | image
image | image

## Pull Request Checklist

 - [x] Ask the contributors if a similar effort is already in process or has been solved.
 - [x] Review the [contribution guidelines](https://github.com/dialpad/dialtone/blob/staging/.github/CONTRIBUTING.md).
 - [ ] Use `staging` as your pull request's base branch. (All PRs using `production` as its base branch will be declined).
 - [x] Ensure all `gulp` scripts successfully compile.
 - [ ] Update, remove, or extend all affected documentation.
 - [ ] Ensure no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).

### CODE REVIEW

**Review by francisrupert — COMMENTED** [review]
@juliodialpad,

Looking great so far. I'll keep pouring over it, but it definitely looks like it was pretty straight-forward.

A forthcoming task will be logged to reconcile `font-weight` (in reaction to your "Changes noticed"). Will likely be Regular, Medium, Semibold, Bold.

File: lib/build/less/variables/typography.less [hunk 1]
```diff
@@ -18,9 +18,9 @@
 //  ============================================================================
 //  $   FONT FAMILY
 //  ----------------------------------------------------------------------------
-@define-ff-custom:                         @ff-inter;
-@define-ff-custom-italic:                  @ff-inter-italic;
-@define-ff-custom-name:                    'Inter';
+@define-ff-custom-name:                    -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, 'Helvetica Neue',
+                                           Arial, 'Noto Sans', sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji',
+                                           'Segoe UI Symbol', 'Noto Color Emoji';
 @define-ff-mono:                           @ff-roboto-mono;
```
Comment (francisrupert) [inline lib/build/less/variables/typography.less]: Update `Segoe UI` with the `ascent-override` adjustment outlined in [DT-644](https://dialpad.atlassian.net/browse/DT-644), solution linked in there.

Comment (juliodialpad) [reply lib/build/less/variables/typography.less]: Done, had a hard time checking if it was working correctly, but seems it is now.

**Comment by francisrupert** [comment]
@juliodialpad,

While we're at it, let's go ahead and update our Mono from `Roboto` to `SFMono`, including its equivalent weights, i.e. in `less/variables/typography.less`...

```less
@ff-sf-mono: {
  400: SFMono-Regular;
  700: SFMono-Bold;
};
```

**Review by francisrupert — APPROVED** [review]
Perfect. The specific stack for Mono might change, but this is 90% there. Approved!

File: lib/build/less/variables/typography.less [hunk 2]
```diff
@@ -167,13 +167,10 @@
     lh-loose:                               @lh-loose;
     lh-looser:                              @lh-looser;
 
-    fw-thin:                                @fw-thin; // Deprecated, but used as shim
-    fw-light:                               @fw-light; // Deprecated, but used as shim
     fw-normal:                              @fw-normal;
     fw-medium:                              @fw-medium; // This is for marketing fonts
     fw-semibold:                            @fw-semibold; // This is for marketing fonts
```
Comment (francisrupert) [inline lib/build/less/variables/typography.less:172]: Remove the `// This is for marketing fonts`, no longer true.

Comment (francisrupert) [reply lib/build/less/variables/typography.less:172]: Hmm, added this comment in wrong PR. Ignore. Moving to other one.

**Pull request merged**